### DISCUSSION TITLE
Sometimes capacity_resume is ignored

### DISCUSSION THREAD

**Discussion by nylimited** [VR-25/acc#172] [General]
Hi everyone,

I could use some helps or suggestions if anyone has ideas.  I am running ACC on a OnePlus N200 and it worked great for a while but at some point it started to ignore the "resume" percentage. As an example, let's say I set _resume=70_ and _pause=80_. ACC always stops charging at 80%. Totally reliable. But, if I plug the device in at, say, 75% charging should not begin if I understand things correctly. 

I cannot prove this yet but it **_seems_** that updating the ROM (using LineageOS 19.1 which tends to update weekly) somehow breaks the resume function. No idea if this is actually possible (the /data/adb directories should not be touched by an update) or if it is a coincidence. Uninstalling and reinstalling ACC, sometimes a few times, eventually fixes the problem.  I have attempted to set/reset config values via ACCA.APK, via ACC in Termux and also via RootExplorer's built-in text editor. I suspect the config file is okay.

Can anyone give me a hint what I might be missing and how I could fix this little issue? Any ideas would be appreciated.

Cheers!

**Comment by VR-25** [comment]
Does this happen with the latest release as well?
